atd-vzd/migrations/migration_atd_txdot_crashes_2023_06_26--1622.sql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE atd_txdot_crashes
-ADD COLUMN in_austin_full_purpose boolean GENERATED ALWAYS AS (is_crash_in_austin_full_purpose(position, city_id)) STORED NOT NULL;
```

Comment:
realized this needs to be nullable, was having some error issues when testing adding a new crash record that had null for lat and long. already made the column nullable in staging/prod so this just reflects those changes

Reply:
awesome - just noting that your change was reflected in the giant migration™️ 🐘